packages/fxa-payments-server/src/lib/amplitude.ts

```diff
@@ -93,12 +95,15 @@ const normalizeEventProperties = (eventProperties: EventProperties) => {
     plan_id = undefined,
     productId = undefined,
     product_id = undefined,
+    paymentProvider = undefined,
+    payment_provider = undefined,
```

Comment:
Based on the comment above this function:
```
// Accepting both planId & plan_id allows us to directly use a Plan type
// as EventProperties
```
It doesn't seem like we'd need to have the snake case version of `paymentProvider` in various places?

Reply:
Yeah, let's rely on the type system to complain when someone try to pass a `payment_provider` property.